pkg/cumuluslog/cumuluslog.go

```diff
@@ -0,0 +1,94 @@
+package cumuluslog
```

Comment:
why cumulus? there is nothing related to cumulus

Reply:
This package is created to provide structured logs for cumulus only. So I think it's needed to highlight its use case, but mb it can be called smth like cumulsurllog.